docs/docs/events/index.md

```diff
@@ -0,0 +1,71 @@
+# Events
+
+Events refer to the activity in a system that tracee monitors. There are two main types of events, built-in events that are part of Tracee and custom events, which references the way users define other events that Tracee should monitor. 
+
+As part of built-in events, there are six types of events:
+
+* syscalls 
+* network 
+* security 
+* lsm 
+* containers 
+* misc
+
+This section documents all of the different events that Tracee exposes.
+
+## Defining events
+
+Tracing the `execve` events in a [policy](../policies/index.md):
+
+```
+apiVersion: tracee.aquasec.com/v1beta1
+kind: Policy
+metadata:
+	name: sample-policy
+	annotations:
+		description: traces execve events
+spec:
+	scope:
+	  - global
+	rules:
+	  - event: execve
+```
+
+If no event is passed with [filters] or [policies], tracee will start with a set of default events.
+
+Please head over to the [Tracee usage](../usage/index.md) documentation for more information on configuring events.
+
+### Event Sets QUESTION
+
+Events can be part of a set. For example, `default`, `network_events`, `syscalls`. 
+We can ask Tracee to trace a full set, or sets, instead of passing event by event, for example:
+
+```
+apiVersion: tracee.aquasec.com/v1beta1
+kind: Policy
+metadata:
+	name: sample-policy
+	annotations:
+		description: traces execve events
+spec:
+	scope:
+	  - global
+	rules:
+	  - event: syscalls
+```
+
+## Read in AVD
```

Comment:
we can remove this now that all events are documented here and actually AVD is out of date